src/cloud/architecture/scaled-architecture.md

```diff
@@ -31,11 +31,11 @@ You can further optimize the performance of the core tier by routing traffic bas
 
 ### Web tier scaling
 
-There are 3 nodes (web nodes) for php-fpm processing and web traffic. In addition to vertical scaling by increasing power and memory, the web tier can scale horizontally by adding extra web servers to an existing cluster when constricted at the PHP level.
+There are 3 nodes (web nodes) for processing requests and web traffic: **php-fpm** and **NGINX**. In addition to vertical scaling by increasing power and memory, the web tier can scale horizontally by adding extra web servers to an existing cluster when constricted at the PHP level.
 
 ![Web tier scaling]
 
-This complements the vertical scaling provided by the core tier. As the core tier scales in size and power to accommodate a growing database and increase in service usage, the web tier scales in size and power, and even instances, to accommodate growing process and traffic requirements.
+This complements the vertical scaling provided by the core tier. As the core tier scales in size and power to accommodate a growing database and increase in service usage, the web tier scales in size and power, and even instances, to accommodate growing process requests and traffic requirements.
```

Comment:
to accomodate growing process requests and traffic requirements --- sounds a little awkward.  Maybe ... accomodate an increase in process requests and higher traffic requirements...